### ISSUE TITLE
Wrong files installed under bin/

### ISSUE THREAD

**Issue by yurivict** [PCMSolver/pcmsolver#116]
Related to this item in the changelog:
* Moved python scripts from bin/ to bin/pcmsolver/ and introduced bin/pcmsolver/__init__.py

Version 1.1.12 on FreeBSD 11.1 now installs files under bin/ that don't make sense:
```
bin/pcmsolver/__init__.py
bin/pcmsolver/codata.py
bin/pcmsolver/docopt.py
bin/pcmsolver/getkw.py
bin/pcmsolver/pcmsolver.py
bin/pcmsolver/plot_cavity.py
bin/pcmsolver/pyparsing.py
bin/run_pcm
```

The previous version installed files that look reasonable:
```
bin/codata.py
bin/docopt.py
bin/getkw.py
bin/pcmsolver.py
bin/plot_cavity.py
bin/pyparsing.py
bin/run_pcm
```

```bin``` directory isn't supposed to have subdirectories, because executables there are supposed to be run through path resolution.

Python files are expected to be installed into lib/pythonN.N/site-packages/pcmsolver/...

```bin/pcmsolver/__init__.py``` is obviously a stray file when placed into bin/

Found this while trying to update the FreeBSD port.

**Comment by robertodr** [comment]
Apologies for breaking packages. This change came about with #110 and addressed this issue #40, but it seems we botched it. We'll have another look, but I am afraid this is low priority for the moment.

**Comment by robertodr** [comment]
@yurivict This was fixed in #121 We'll be minting a release candidate soon and update Psi4 accordingly.
